website/pages/docs/drivers/exec.mdx

```diff
@@ -77,6 +77,16 @@ task "example" {
 }
 ```
 
+## Capabilities
+
+| Feature | Implementation |
+| --- | --- |
+| SendSignals | true  |
+| Exec | true |
+| FSIsolation | FSIsolationImage |
```

Comment:
Should be chroot: https://github.com/hashicorp/nomad/blob/v0.12.0/drivers/exec/driver.go#L82